### PR TITLE
Settings Support

### PR DESCRIPTION
**Pull request by Snazzah**

### CODE REVIEW

File: rubot.rb [hunk 1]
```diff
@@ -110,6 +113,41 @@ def handle(event_type, payload)
...
+    case payload.action
+    when "closed"
+      return false
+    when "unlabeled"
+      return false 
+    when "unassigned"
+      return false
+    end
+  end
+  if not $config["showprs"] and event_type = "pull_request"
```
Comment (meew0) [inline rubot.rb]: All these if statements seem fairly messy, could you combine them into a loop and a hash? (Also, you need to use `==` for comparison - how did this ever work?)

File: rubot.rb [hunk 2]
```diff
@@ -123,12 +161,14 @@ def handle(event_type, payload)
   channels = $links[repo_name]
   channels.each do |e|
     response = handle(event_type, payload)
-    if response
-      bot.send_message(e, "**#{repo_name}**: **#{payload['sender']['login']}** " + response)
-    else
-      puts %(Got a "#{event_type}" event for repo #{repo_name} that is not supported - ignoring)
+    if check_config(event_type, payload(
```
Comment (meew0) [inline rubot.rb]: You got a typo here

**Comment by meew0** [comment]
Thanks for the effort, but in the future, could you test the code you've written before making a PR, to avoid obvious typos like those? Or at least run it through RuboCop, which I don't have set up on this repo

File: rubot.rb (hunk 1 above)
Comment (Snazzah) [inline rubot.rb]: How would i do so?

Comment (meew0) [inline rubot.rb]:
``` rb
{
  "showprs" => "pull_request",
  # ...
}.each do |config_type, event_name|
  # the check in here
end
```

**Pull request closed without merging**